src/EnergyPlus/GeneratorDynamicsManager.hh

```diff
@@ -75,15 +74,12 @@ namespace GeneratorDynamicsManager {
                                      Real64 &ElecLoadProvided,                     // power allowed
                                      DataGenerators::OperatingMode &OperatingMode, // operating mode
                                      Real64 &PLRforSubtimestepStartUp,             // part load ratio for switch to normal from start up
-                                     Real64 &PLRforSubtimestepShutDown,            // part load ratio for switch from cool down to other
-                                     bool const FirstHVACIteration                 // True is this is first HVAC iteration
+                                     Real64 &PLRforSubtimestepShutDown             // part load ratio for switch from cool down to other
     );
 
     void ManageGeneratorFuelFlow(EnergyPlusData &state,
                                  GeneratorType const GeneratorType, // type of Generator
```

Comment:
Do you really need to pass this as a paremeter? Can you not get it internally using `GeneratorNum`?

Reply:
Do `we` really ...

Reply:
From what I can tell, the GeneratorType is not saved and is passed as an argument.

    GeneratorDynamicsManager::ManageGeneratorControlState(state,
                                                          GeneratorType::MicroCHP,

Reply:
Pfffft.